ai-eval-service/main.py

```
@@ -235,15 +183,15 @@ async def evaluate_batch(request: BatchEvaluateRequest) -> List[BatchEvaluateIte
     except HTTPException:
         raise
     except ValueError as e:
-        logger.warning(f"Validation error in batch evaluation: {e}", exc_info=False)
+        logger.warning(f"Validation error: {e}", exc_info=False)
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)